src/Particle/ParticleSet.cpp

```diff
@@ -43,10 +43,12 @@ template<> int ParticleSet::Walker_t::cuda_DataSize = 0;
 
 void add_p_timer(std::vector<NewTimer*>& timers)
 {
-  timers.push_back(new NewTimer("ParticleSet::makeMove",timer_level_fine)); //timer for MC, ratio etc
-  timers.push_back(new NewTimer("ParticleSet::makeMoveOnSphere",timer_level_fine)); //timer for the walker loop
+  timers.push_back(new NewTimer("ParticleSet::makeMove",timer_level_fine)); //timer for moves
```

Comment:
The createTimer method on TimerManager will create and add a timer in one call.  So this could be simplified to
timers.push_back(TimerManager.createTimer("ParticleSet::makeMove", timer_level_fine)); // timer for moves
Then there is no need for TimerManager.addTimer(timers[0])